docs/controls/Chip.md

```diff
@@ -0,0 +1,85 @@
+# Chip
+
+## Summary
+
+`Chip` is derived `ToggleButton` that can be used for selection, filters or trigger action selection. `ChipGroup` can be used to display a list of `Chip`. 
+
+## Features
+
+### Chip
+
+| Properties         | Type         | Description                                                | Supported       |
+|--------------------|--------------|------------------------------------------------------------|-----------------|
+| IsCheckable        | bool         | Whether the chip can be checked                            | All platforms   |
```

Comment:
note: When used inside a `ChipGroup`, this property will be overwritten by `ChipGroup`'s `SelectionMode`.